packages/shared_preferences/shared_preferences_macos/README.md

```diff
@@ -0,0 +1,32 @@
+# shared_preferences_macos
+
+The macos implementation of [`shared_preferences`][1].
+
+## Usage
+
+### Import the package
+
+To use this plugin in your Flutter app, simply add it as a dependency in
+your `pubspec.yaml` alongside the base `shared_preferences` plugin.
+
+_(This is only temporary: in the future we hope to make this package an
+"endorsed" implementation of `shared_preferences`, so that it is automatically
+included in your Flutter app when you depend on `package:shared_preferences`.)_
```

Comment:
optional nit: It would be kind of nice to link to a tracking issue here if there is one.

(I also think it would be nice if macos_connectivity had a README like this too, but it's not critical at all.)